packages/actor-scraper/cheerio-scraper/README.md

```diff
@@ -246,19 +246,16 @@ visit the [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/J
 
 - ##### **`Apify: Object`**
 
-  A reference to the main namespace of [Apify SDK](https://sdk.apify.com/docs/api/apify).
+  A reference to the [Actor](https://apify.github.io/apify-sdk-js/api/apify/class/Actor) object from [Apify SDK](https://apify.github.io/apify-sdk-js/api/apify/).
   This is equivalent to:
 
   ```javascript
-  const Apify = require('apify');
+  const { Actor: Apify } = require('apify');
```

Comment:
i would go for ESM examples, and use `Actor` directly without aliasing. wdyt @mnmkng?

Reply:
yes